docs/_snippets/webhook-signature-validation.mdx

```diff
@@ -0,0 +1,18 @@
+```typescript
+
+const signature = req.headers["x-metriport-signature"];
+  const metriportApi = new MetriportMedicalApi(apiKey, {
+    baseAddress: apiUrl,
+  });
+  try {
+    if (metriportApi.verifyWebhookSignature(whKey, req.body, signature)) {
+      console.log(`Signature verified`);
+    } else {
+      console.log(`Signature verification failed`);
+    }
+  } catch (error) {
+    console.error(`Error verifying signature: ${error}`);
+    return res.status(500).send({ error: `Error verifying signature: ${error}` });
+  }
```

Comment:
Why the `try/catch` if the `verifyWebhookSignature()` function returns true/false to success/failure?

Reply:
Just saw the implementation can throw, but this is not ideal DX. Ideally it would either return true/false or throws on error.

Reply:
remove the error throw - unnessecary with forcing the type of signature to be a string